index.html

```diff
@@ -49,4 +49,4 @@ <h3><em>Work with us its. </em> <br></h3>
      <p><em>©wezesha</em></p>
    </footer>
  </body>
-</html>
```

Comment:
## Best practices
- Its considered good coding practice to have a new line at the end of the file it enhances readability

Reply:
While you did a good job implementing all the required comments :+1: please ensure to always state explicit reasons why you requested a change; it might be obvious to you and not so to others.